src/main/java/hudson/remoting/Request.java

```diff
@@ -192,10 +192,10 @@ final RSP call(Channel channel) throws EXC, InterruptedException, IOException {
                     // ignore the I/O error
                 }
 
-                Object exc = response.exception;
+                Throwable exc = response.exception;
```

Comment:
`response.exception` was always of type `Throwable`, so this was just unnecessary.